charts/fleet/charts/gitjob/values.yaml

```diff
@@ -15,3 +15,12 @@ global:
 
 # comma separated list of domains or ip addresses that will not use the proxy
 noProxy: 127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,.svc,.cluster.local
+
```

Comment:
Can you remove this change? I would like to get this updated along with a new tag in gitjob next time so it is easy to track

Reply:
Yes